template.md

```diff
@@ -0,0 +1,44 @@
+# Title of the enhancement proposal
+
+<!-- Add link to issue/epic if available -->
+
+## Summary
+
+<!-- Add brief summary of enhancement proposal. -->
+
+## Proposal
+
+<!-- Detailed information about the enhancement proposal. Add any subsections if needed. -->
+
+## Design Details
+
+<!-- Detailed design of the proposal. This should include any yamls, CRDs, data structures, APIs etc. Add/remove any subsections
+if needed -->
+
+### Pros
+<!-- Benefits of going with this proposal  -->
+
+### Cons
+<!-- Drawbacks of going with this proposal. -->
+
+### Backward Compatibility
+
+<!-- Any backward compatibility concerns -->
+
+### Alternatives
+
+<!-- Any alternatives to consider with pros and cons, if applicable. -->
+
+## External Dependencies
+
+<!-- Any external dependencies this proposal may have -->
+
+## User Impact
+
+<!-- Optional. Any impact on users this change may have. -->
+
+## Owners
+
+<!-- Optional. List of owners for this enhancement proposal and its implementation -->
+* Owner 1 Name <owner1@email.com>
+* Owner 2 Name <owner2@email.com>
```

Comment:
IDK if this makes any sense to put here.

Reply:
> IDK if this makes any sense to put here.

Added it based on similar field used in such templates for previous projects. Wasn't sure if makes sense for us, added so could decide in review.